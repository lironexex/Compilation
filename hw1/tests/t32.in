// other tokens:
import UIKit
var num : Double = 1.0e+6
String Int UInt Float Bool Character
char string int uint float bool character
let y = 3
func
nil
if 
else
return;
,
() {} [] = == != < > <= >=
|| && %
true false
->
:
