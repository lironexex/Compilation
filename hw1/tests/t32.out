1 COMMENT 1
2 IMPORT import
2 ID UIKit
3 VAR var
3 ID num
3 COLON :
3 TYPE Double
3 ASSIGN =
3 DEC_REAL 1.0e+6
4 TYPE String
4 TYPE Int
4 TYPE UInt
4 TYPE Float
4 TYPE Bool
4 TYPE Character
5 ID char
5 ID string
5 ID int
5 ID uint
5 ID float
5 ID bool
5 ID character
6 LET let
6 ID y
6 ASSIGN =
6 DEC_INT 3
7 FUNC func
8 NIL nil
9 IF if
10 ELSE else
11 RETURN return
11 SC ;
12 COMMA ,
13 LPAREN (
13 RPAREN )
13 LBRACE {
13 RBRACE }
13 LBRACKET [
13 RBRACKET ]
13 ASSIGN =
13 RELOP ==
13 RELOP !=
13 RELOP <
13 RELOP >
13 RELOP <=
13 RELOP >=
14 LOGOP ||
14 LOGOP &&
14 BINOP %
15 TRUE true
15 FALSE false
16 ARROW ->
17 COLON :
